console/my-account/concepts.mdx

```diff
@@ -87,7 +87,7 @@ Multifactor authentication (MFA) is any form of verification that requires two f
 
 ## Organization Quotas
 
-Every [Organization](#organization) has quotas, which are limits on the number of Scaleway resources they can use. Quotas are per product (eg Instances) and product type (eg GP1-L instance). Quotas are designed to prevent abuse, and can be viewed on the [Organization Account page](https://console.scaleway.com/account/organization/profile) of the console.
+Every [Organization](#organization) has quotas, which are limits on the number of Scaleway resources they can use. Quotas are per product (e.g. Instances) and product type (e.g. GP1-L instance). Quotas are designed to prevent abuse, and can be viewed on the [Organization Account page](https://console.scaleway.com/account/organization/profile) of the console.
```

Comment:
```suggestion
Every [Organization](#organization) has quotas, which are limits on the number of Scaleway resources they can use. Quotas are per product (e.g. Instances) and product type (e.g. GP1-L Instance). Quotas are designed to prevent abuse, and can be viewed on the [Organization dashboard](https://console.scaleway.com/account/organization/profile) of the console,.
```